lando/worker/cwlworkflow.py

```diff
@@ -3,9 +3,90 @@
 """
 from __future__ import print_function
 import os
+import shutil
 import urllib
+import datetime
+import json
 from subprocess import PIPE, Popen
 from lando.exceptions import JobStepFailed
+from lando.worker.cwlreport import create_workflow_info, CwlReport
+
+
+RUN_CWL_COMMAND = "cwl-runner"
+RUN_CWL_OUTDIR_ARG = "--outdir"
+
+REPORT_FILENAME = 'Bespin-Report.txt'
+
+LOGS_DIRECTORY = 'logs'
+JOB_STDOUT_FILENAME = 'cwltool-output.json'
+JOB_STDERR_FILENAME = 'cwltool-output.log'
+JOB_DATA_FILENAME = 'job-data.json'
+
+WORKFLOW_DIRECTORY = 'workflow'
+WORKFLOW_FILENAME = 'workflow.cwl'
+JOB_ORDER_FILENAME = 'workflow.yml'
```

Comment:
CWL supports both formats. The CWL guide shows everything as YAML (JSON is valid YAML). We are storing it as JSON in the database field when generating it from a questionnaire.